### ISSUE TITLE
2.8.0-rc1 pre-release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2862]
- #2839 
- #2837 
- #2853 
- #2850 
- #2844
- #2841 
- #2858 
- #2859 
- #2840 
- #2846

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick 35e924e575a36794f35d43deb534cafb255c82dd
git cherry-pick 0e449cb2ab1846615c38517f020f072ead967929
git cherry-pick 491287a518ebd45b7c59c5fd8e5b62dc019faa12
git cherry-pick a805be1f5b55d08d6f0e4e18274b3ef0678f4458
git cherry-pick 12ff971d579643837af2e3cc35ce75a1984fe10d
git cherry-pick 047a8c40dc85ccd3fdfa76303ed02a9f890db5ac
git cherry-pick 530cc2bac285bdefda419b5bc77e5fd0b80b22b9
git cherry-pick 9a7bbc4a79dd8632a1de15177ab808c323329477
git cherry-pick 78000928f481a13272baf020b7692aea3b9f0a97
git cherry-pick 48ca576de5dce188c1d6c1363c066968cb24be32
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.8.0-beta2..HEAD --pretty='format:- %s'`
```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2.  Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

- Closes #2845: Ensure legacy Quickstart block_content view is replaced by core view. (#2846)
- Update drupal/blazy requirement from 2.20.0 to 2.21.0 (#2840)
- Update drupal/token requirement from 1.12.0 to 1.13.0 (#2859)
- Update drupal/webform requirement from 6.2.0-rc1 to 6.2.0-rc3 (#2858)
- #2827: Use stable embed release and final entity embed patch for CKEditor5 support. (#2841)
- Closes #2843: Address blazy module status report error in 2.8.x. (#2844)
- Closes #2849 CKEditor preview styling overridden by AZ Barrio styling in 2.8 beta2 (#2850)
- Fixes #2851 Cannot Save Curation Options (#2853)
- Closes #2836: Fix PR branch naming convention in prepare-for-minor-release workflow. (#2837)
- Closes #2838 Fix failing create-release workflow. (#2839)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created
https://github.com/az-digital/az_quickstart/releases/tag/2.8.0-rc1
